src/utils/index.js

```diff
@@ -1,8 +1,8 @@
 /** @format */
+
 export * from './html'
 export * from './css'
 export * from './rtl'
 export * from './text'
-export * from './animations'
 export * from './timings'
-export * from './withAnimatedCloseHOC'
+export * from './animations/index'
```

Comment:
you don't need `/index` here I think

Reply:
For some reason `./animations` didn't get work on in my dev env. Try at your end if it works.